### DISCUSSION TITLE
Feature Request for Menu Item options (attributes, icons and subtitle)

### DISCUSSION THREAD

**Discussion by viocassel** [joomla/joomla-cms#14383] [4. Rejected Features]
How about to add options to the Menu Item -> Link Type: Link Attribute, Link Icon and Menu Subtitle?

Adding Link Attribute, Menu Subtitle and Link Icon (**Image**) from Media Manager

![85156c87117e45adab31166cd034457e](https://cloud.githubusercontent.com/assets/3341562/23612508/ba7399f2-028c-11e7-93bc-98d52c97388a.jpeg)

Or adding Link Icon like **HTML**

![82edb1200a48462da2579ac6990870e6](https://cloud.githubusercontent.com/assets/3341562/23612436/7c88fb96-028c-11e7-96f6-0e7415711e5e.jpeg)

Result like this... In this case I used [UIkit 3 framework](https://getuikit.com/).

![e7b7fd3f9db54fc3ae329a57c939df2b](https://cloud.githubusercontent.com/assets/3341562/23612842/0d94cf7e-028e-11e7-91e2-b1adb715ec36.jpeg)
```
<nav class="uk-navbar">
    <ul class="uk-navbar-nav">
        <li>
            <a class="joomla" data-attr="joomla" href="#" >
                <span uk-icon="icon: joomla"></span>
                Home
                <div class="uk-navbar-subtitle">Joomla! Start</div>
            </a>
        </li>
        <li>...</li>
    </ul>
</nav>
```

No matter what icon libraries or front-end framework you use.

**Menu Subtitle** option should be moved to some other tab... maybe... 

Maybe someone would be able to implement this idea or someone sees this functionality as something else? 😉

**Comment by proweb** [comment]
+1

**Comment by HLeithner** [comment]
The Joomla Maintainer Team decided at the 25.02.2026 meeting to reject this feature request because we try to avoid more parameter fields if possible. A better solution would be to add custom fields instead of further parameters.
